Makefile

```diff
@@ -140,3 +140,7 @@ benchmarks-comp:
 benchmarks-stargz:
 	@echo "$@"
 	@cd benchmark/stargzTest ; GO111MODULE=$(GO111MODULE_VALUE) go build -o ../bin/StargzTests . && sudo ../bin/StargzTests $(COMMIT) ../singleImage.csv 10 $(STARGZ_BINARY)
+
+benchmark-collect-file-access-patterns:
```

Comment:
Why made a new target instead of making it a command line option of``make benchmark``?